src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ReplayDataPlaybackController.cs

```diff
@@ -613,8 +617,36 @@ private void PlayInputs(Dictionary<int, RecordedGameObjectState> priorStates, Di
             _mouseQueue.RemoveAll(a => a.IsDone);
         }
 
+#if ENABLE_LEGACY_INPUT_MANAGER
```

Comment:
This is a flag that is built into the Unity engine (since around version 2019.2, as far as I can tell). https://docs.unity3d.com/2021.3/Documentation/Manual/PlatformDependentCompilation.html

Reply:
So, for a project that was already using the legacy input manager, it will be automatically enabled. Note that it is possible for both ENABLE_LEGACY_INPUT_MANAGER and ENABLE_INPUT_SYSTEM to be enabled together if the project happens to be using both systems.